scripts/jenkins/mkcloud/openstack-mkcloud.builder.sh

```diff
@@ -11,6 +11,11 @@ mkdir -p $artifacts_dir
 touch $artifacts_dir/.ignore
 export log_dir=$artifacts_dir/mkcloud_log
 
+if [ -d $WORKSPACE/automation-git ] ; then
```

Comment:
For this to work use https://docs.openstack.org/infra/jenkins-job-builder/wrappers.html#wrappers.workspace-cleanup%20(pre-build) and remove line 4-6 above, otherwise this is will have been deleted.